src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -107,16 +109,11 @@ class BlockingProcess(
     }
 
     try {
-      val stdin =
-        context.actorOf(OutputStreamSubscriber.props(process.getOutputStream), "stdin")
+      val stdin = Sink.outputStream(process.getOutputStream)
+      val stdout = Source.inputStream(process.getInputStream)
+      val stderr = Source.inputStream(process.getErrorStream)
```

Comment:
> The Actor within the Source.inputStream() is created by calling materializer.actorOf - so my understanding this means this Actor is the child of where the Source is materialized. I'm not 100% sure if my understanding is corrrect.

The exact rule is this:

When the ActorMaterializer is created, it uses the ActorRefFactory in scope.
It can be an ActorSystem, or it can be an ActorContext if you're inside an Actor.
It then uses that factory to create the `StreamSupervisor`, as a child of that context,
so if it was an ActorContext, it will be a child of your Actor - the Supervisor is then the parent of the 
actual Source/Sink/Flow executing Actors.

So to answer if they're children, one has to see where the materializer was created.
In Actors you should use `with ImplicitMaterializer`